mobile/modules/users/less/_login.less

```diff
@@ -125,19 +196,43 @@
     border-bottom: 1px solid white;
     padding: 2.6rem 0 1rem 0;
     margin-bottom: 2.5rem;
+
     .placeholder(@color-bali-hai-blue-s24);
 
     &.ng-invalid:not(.ng-pristine) {
       color: @login-error-color;
       border-bottom: 1px solid @login-error-color;
 
+      .placeholder(@login-error-color);
+
       ~ .error-message .help-block {
         color: @login-error-color;
       }
     }
 
     &--password{
-      margin-bottom: 0;
+      letter-spacing: .25rem;
+      font-size: 1.5em;
+
+      &::-webkit-input-placeholder {
+        font-size: 1.4rem;
+        letter-spacing: 0;
+      }
+
+      &::-moz-placeholder {
+        font-size: 1.4rem;
+        letter-spacing: 0;
+      }
+
+      &:-moz-placeholder {
+        font-size: 1.4rem;
+        letter-spacing: 0;
+      }
+
+      &:-ms-input-placeholder {
+        font-size: 1.4rem;
+        letter-spacing: 0;
```

Comment:
in this case we could take out common styles

&::-webkit-input-placeholder,
&::-moz-placeholder,
etc. {
     font-size: 1.4rem;
     letter-spacing: 0;
}